benches/index.ts

```diff
@@ -1,18 +1,41 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node
 
 import fs from 'fs';
+import path from 'path';
 import si from 'systeminformation';
 import WorkerManagerBench from './WorkerManager';
```

Comment:
Make this `import workerManager from './worker_manager.ts`.

Reply:
The new metrics use lower case names for the reports, which I retain as part of the file name.